dex/TAG

```diff
@@ -1 +1 @@
-2.37.0.3
+2.37.0.4
```

Comment:
I think dex's TAG doesn't have to be updated.

Reply:
Ah, I see. You updated it to confirm the workflow working correctly.

Reply:
We've confirmed the actions working correctly, so now, the TAG should be reverted :)